src/components/Flex/Flex.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useRef, forwardRef, useMemo } from "react";
+import cx from "classnames";
+import useMergeRefs from "../../hooks/useMergeRefs";
+import Clickable from "../Clickable/Clickable";
+import { FLEX_JUSTIFY, FLEX_GAPS, FLEX_DIRECTIONS, FLEX_ALIGN } from "./FlexConstants";
+import classes from "./Flex.module.scss";
+import VibeComponentProps from "src/types/VibeComponentProps";
+import VibeComponent from "src/types/VibeComponent";
+
+export type FlexProps = VibeComponentProps & {
+  /** class name to be add to the wrapper */
+  className?: string;
+  /** id to pass to the element */
+  id?: string;
+  style?: object;
+  direction?: FLEX_DIRECTIONS;
+  elementType?: React.ElementType;
+  wrap?: boolean;
+  children?: React.ReactElement;
+  justify?: FLEX_JUSTIFY;
+  align?: FLEX_ALIGN;
+  gap?: FLEX_GAPS | number;
+  ariaLabel?: string;
+  tabIndex?: number;
+  /** onClick function - MouseEvent */
+  onClick?: React.MouseEventHandler;
```

Comment:
Hi @piotrski
the onClick could also be from a keyboard event, please add it as well